mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/MantisWorker.java

```diff
@@ -22,12 +22,16 @@
 import io.mantisrx.runtime.Job;
 import io.mantisrx.server.core.BaseService;
 import io.mantisrx.server.core.Service;
+import io.mantisrx.server.core.WrappedExecuteStageRequest;
+import io.mantisrx.server.master.client.ClassLoaderHandle;
 import io.mantisrx.server.master.client.HighAvailabilityServices;
 import io.mantisrx.server.master.client.HighAvailabilityServicesUtil;
 import io.mantisrx.server.master.client.MantisMasterGateway;
+import io.mantisrx.server.master.client.SinkSubscriptionStateHandler;
```

Comment:
should MantisWorker move to the agent module as well?

Reply:
See my commit description. MantisWorker is Job aware. It supports passing a Job object directly. It must depend on mantis-runtime. So it's not possible to move to agent.